### PR TITLE
Completing step 2 of updating loki-build-image

### PR DESCRIPTION
**Pull request by dannykopping**
**What this PR does / why we need it**:

Updates the `loki-build-image`  and corrects mistake in https://github.com/grafana/loki/pull/7793 where I overwrote the `0.24.2` image.

[Step 1](https://github.com/grafana/loki/pull/7795)

**Which issue(s) this PR fixes**:
N/A

**Special notes for your reviewer**:
Followed my docs and found one step missing.

**Checklist**
- [ ] Reviewed the `CONTRIBUTING.md` guide
- [ ] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/upgrading/_index.md`

### CODE REVIEW

**Review by chaudum — APPROVED** [review]

**Review by DylanGuedes — APPROVED** [review]
lgtm, just a small nit. Do you mind also rephrasing the PR description? Since it will  listed  after we merge this PR

edit: damn you're fast. The current PR description looks good enough, thank you.

File: docs/sources/maintaining/release-loki-build-image.md
Comment (dannykopping) [reply docs/sources/maintaining/release-loki-build-image.md:27]: I think it's OK as is, but I mainly don't want to wait for CI again :crying_cat_face:

**Pull request merged**
